Removed cartridge-java from dependencies (#71)
tarantool/cartridge-java-testcontainers: CHANGELOG.md

# Changelog

## [0.6.0] - 2023-05-17

> BREAKING CHANGES

Reviewer: Not all of them are breaking changes. Look how we did it cartridge-java 
![image](https://github.com/tarantool/cartridge-java-testcontainers/assets/22258443/240679f9-96bb-4b4e-85ec-c6450a09b3e5)

## [Unreleased]
- Fix problem if topology isn't applied correctly
- Bump testcontainers to 1.18.0
- Move rocks building to the build phase
- **[breaking change]** Remove io.tarantool.cartridge-driver dependency

> - **[breaking change]** Update executeScript and executeCommand methods to execute code viva execInContainer  

Reviewer 2: ```suggestion
- **[breaking change]** Update executeScript and executeCommand methods to execute code via execInContainer  
```

>   (now it returns yaml string in Container.ExecResult not CompletableFuture)

Reviewer 2: ```suggestion
  (now it returns **yaml** string in Container.ExecResult instead of CompletableFuture)
```

- Fix problem if topology isn't applied correctly
- Bump testcontainers to 1.18.0
- Move rocks building in build phase
- Bump testcontainers to 1.18.0

> - Move rocks building in build phase

Reviewer 3: ```suggestion
- Move rocks building to the build phase
```
